terraform/modules/cluster/mig-with-container/cloudinit/templates/aiinfra_start_container.yaml.template

```diff
@@ -0,0 +1,31 @@
+- path: /etc/systemd/system/aiinfra_start_container.sh
+  permissions: 0755
+  owner: root
+  content: |
+    #!/bin/sh
+    docker run \
+      --detach \
+      --hostname $(hostname) \
+      --ipc host \
+      --name aiinfra \
+      --network host \
+      --privileged \
+      --restart always \
+      ${docker_volume_flags} \
+      ${docker_run_options} \
+      ${docker_image} ${docker_cmd}
```

Comment:
when container is null, this docker_image and docker_cmd would be empty. Would not that throw docker error ? Should we have a condition to run docker command only when docker_image is available ?